next-hardhat/.gitignore

```diff
@@ -38,13 +38,11 @@ yarn-error.log*
 *.tsbuildinfo
 next-env.d.ts
 
-# nargo
-target
-contract
+# contract
 
 # hardhat
 cache
-artifacts
+# artifacts
```

Comment:
do we want to track the artifacts? I dont think we need to since they are generated files.

Reply:
```suggestion
artifacts
```